index.js

```diff
@@ -472,9 +472,9 @@ class PDFDocumentWithTables extends PDFDocument {
           }
 
           // 24.1 is height calc title + subtitle
-          titleHeight = !lockAddTitles ? 24.1 : 0; 
+          titleHeight = !lockAddTitles ? 24.1 : 0;
           // calc if header + first line fit on last page
-          const calc = startY + titleHeight + firstLineHeight + this.headerHeight + safelyMarginBottom// * 1.3;
+          const calc = startY + titleHeight + firstLineHeight + this.headerHeight + options.marginBottom// * 1.3;
```

Comment:
Are you not already adding the marginBottom to the safelyMarginBottom. Wouldnt you want both to be considerated. ergo not changing this line